Scripts/Connector/Sample connector restore script.ps1

```diff
@@ -0,0 +1,34 @@
+# This script restores a connector that has been backed up.
+# To backup a Connector look at "Sample connector backup script.ps1"
+
+# RUN THIS SCRIPT IN AN ELEVATED POWERSHELL WINDOW
+
+# BEFORE RUNNING THESE COMMANDS PERFORM THE FOLLOWING STEPS:
+# 1) Copy the C:\ConnectorBackup folder (generated by the backup script) to the machine in which you will restore it
+# 2) Install the latest Connector from https://aka.ms/UPConnector (do not register it)
```

Comment:
do they need to launch or log in?

Reply:
I've tried it both launching it and without launching it and it doesn't seem to matter